container/Dockerfile

```diff
@@ -0,0 +1,111 @@
+# BUILDER
+FROM alpine:latest AS builder
+
+#build ARGs
+ARG _RADARE2_GIT_REPO="https://github.com/radareorg/radare2.git"
+ENV RADARE2_GIT_REPO=$_RADARE2_GIT_REPO
+
+ARG _KS_NAV_GIT_REPO="https://github.com/elisa-tech/ks-nav"
+ENV KS_NAV_GIT_REPO=$_KS_NAV_GIT_REPO
+ARG _KS_NAV_GIT_REPO_BRANCH=main
+ENV KS_NAV_GIT_REPO_BRANCH=$_KS_NAV_GIT_REPO_BRANCH
+
+# install build dependecies
+RUN apk add --no-cache autoconf automake build-base git
+RUN apk add --no-cache go
+RUN apk add --no-cache linux-headers
+
+#build app
+WORKDIR /build
+
+# Build radare2
+RUN git clone ${RADARE2_GIT_REPO} /build/radare2 && cd /build/radare2 && git checkout ecd8d00aad289abf8f850b45d5f6c2e6b0c15c0f
+RUN cd /build/radare2 && ./configure && make && make install
+
+# Build ks-nav
+RUN git clone ${KS_NAV_GIT_REPO} /build/ksnav && cd /build/ksnav/kern_bin_db && git checkout ${KS_NAV_GIT_REPO_BRANCH}
+RUN cd /build/ksnav/kern_bin_db && make && make install
+RUN cd /build/ksnav/nav && make && make install
+RUN cd /build/ksnav/navweb && go get -u github.com/go-bindata/go-bindata/... && go install github.com/go-bindata/go-bindata/... && make && make install
+
+
+
+##############################################################################################################################################
+
+# RUNTIME
+FROM alpine:latest
+
+#run ARGs
+ARG _POSTGRES_NAME=only
+ENV POSTGRES_NAME=$_POSTGRES_NAME
+ARG _POSTGRES_DATA_DIR=/var/lib/postgresql/data/
+ENV POSTGRES_DATA_DIR=$_POSTGRES_DATA_DIR
+ARG _POSTGRES_IDENT_FILE=pg_ident.conf
+ENV POSTGRES_IDENT_FILE=$_POSTGRES_IDENT_FILE
+ARG _POSTGRES_CONF_FILE=${POSTGRES_DATA_DIR}/${POSTGRES_NAME}/postgresql.conf
+ENV POSTGRES_CONF_FILE=$_POSTGRES_CONF_FILE
+
+ARG _START_SCRIPT_NAME=/usr/bin/start.sh
+ENV START_SCRIPT_NAME=$_START_SCRIPT_NAME
+
+# install dependencies
+RUN apk add --no-cache postgresql postgresql-contrib postgresql-client
+RUN apk add --no-cache binutils
+RUN apk add --no-cache graphviz
+
+# setup resources
+
+WORKDIR /app
+COPY postgres_conf_template.conf /tmp
+# pg.db.tar.gz is a link, please set it to empty database, or whatever db you have.
+copy pg.db.tar.gz /tmp
```

Comment:
Let's follow the convention and use capital `COPY` here

Reply:
Sure, thanks